__tests__/classes/DOMMatrix.js

```diff
@@ -160,4 +145,158 @@ describe('DOMMatrix class', () => {
     matrix.m42 = 2;
     expect(matrix.f).toBe(2);
   });
+
+  describe(`translate`, function () {
+    it(`should return a new DOMMatrix instance`, function () {
+      const matrix = new DOMMatrix();
+      const translatedMatrix = matrix.translate(100, 100);
+      expect(translatedMatrix instanceof DOMMatrix).toBeTruthy();
+      expect(translatedMatrix === matrix).toBeFalsy();
```

Comment:
This test is techincally correct. I was curious if you might prefer `expect(translatedMatrix).not.toBe(matrix);`?

Reply:
I try to avoid the `not` operator. 
Most of the times using `not` leads to unintended errors (trying to prove something does not exist is impossible logically).
But that's just my philosophy teacher speaking. 
It doesn't matter in this case, so I can change that here.

Reply:
I don't think it's necessary to make you write code you aren't proud of. :)

I happen to be the creator of the testing framework `as-pect` for assemblyscript, which implemented a similar `expect()` api, so this particular idea crossed my mind. In this case, we are simply trying to assert the objects are different, and in the case of simple equality, the `.not` operator works just fine. I was merely questioning why you didn't use it. 

No need to fix it.